Dockerfile

```diff
@@ -130,7 +130,7 @@ RUN apk --update --no-cache add -t build-dependencies \
 
 COPY rootfs /
 
-EXPOSE 8000 514 514/udp
+EXPOSE 8000 514 514/udp 162 162/udp
```

Comment:
TCP port 162 is unnecessary. All SNMP transports are UDP.  Even DTLS is UDP - although that is port 10161/2 - can the sidecar handle encrypted traps?

Reply:
we need to extend the snmptrapd.conf to add credentials for SNMPv2c (authCommunity) (already there but changes need docker image rebuild) and SNMPv3 (createUser) for both USM (Password-based) and TSM (DTLS - certs based), but as that is seldom used, we could start with simple v2c/v3-USM support and extend it later.

Reply:
SNMP isn't udp only. I'm not intrested in SNMPv3 or DTLS so feel free to add that ontop of this PR yourself.

Unencrypted SNMP traps over udp and tcp is MVP for me.